Makefile

```diff
@@ -88,6 +93,18 @@ container-push: ## Push docker container image
 cmd: ## Opens a shell in docker container
 	$(docker) /bin/bash
 
+##@ SPIRE images
+
+.PHONY: spire-images
+spire-images: spire-server-image spire-agent-image
+
+.PHONY: spire-server-image
+spire-server-image: Dockerfile.server
```

Comment:
Maybe a double-sharp-starting-comment with a brief description of the target could be added for the SPIRE images targets. This message will be later parsed and printed when make help is called.

For example :
```
spire-server-image: Dockerfile.server ## Builds a SPIRE server docker image
```

Reply:
Another great catch! Thanks.